packages/fast-components-react-msft/src/media/examples.data.ts

```diff
@@ -0,0 +1,37 @@
+import { ICategoryItemProps } from "@microsoft/fast-development-site-react";
+import { ISnapshotTestSuite } from "@microsoft/fast-jest-snapshots-react";
+import { IManagedClasses } from "@microsoft/fast-jss-manager-react";
+import Media from "./index";
+import { IMediaHandledProps } from "@microsoft/fast-components-react-base";
+
+export default {
+    name: "media",
+    component: Media,
+    data: [
+        {
```

Comment:
Not a requirement, but I wonder if we've tested enough of the variations on these. Can you add more examples which include or don't include certain options, etc? This may help us identify places where our implementation can be improved or may have gaps.